src/pydvl/valuation/types.py

```diff
@@ -29,10 +30,17 @@
 NullaryPredicate = Callable[[], bool]
 
 
+class ValueUpdateKind(str, Enum):
```

Comment:
Maybe a short docstring would be helpful, e.g. what does `DEFAULT` mean, is it context dependend?

Reply:
As discussed, we got rid of `ValueUpdateKind`